lib/cfer/core/stack.rb

```diff
@@ -159,6 +159,17 @@ def to_cfn
       to_h.to_json
     end
 
+    # Includes template code from one or more files, and evals it in the context of this stack.
+    # Filenames are relative to the file containing the invocation of this method.
+    def include_template(*files)
+      calling_file = caller.first.split(/:\d/,2).first
```

Comment:
Huh, I'm not sure how I expected `require_relative` to be implemented, but that definitely wasn't it. :)

Yeah I'm good with this.